### PR TITLE
Fix 230

### PR DESCRIPTION
**Pull request by ben-kaufman**

### CODE REVIEW

File: contracts/vaults/Deposit.sol [hunk 1]
```diff
@@ -40,7 +41,6 @@ contract Deposit is Base {
 
         user.shares += userShares;
         pool.totalShares += userShares;
-        user.rewardDebt = user.shares * pool.rewardPerShare / 1e12;
```
Comment (jellegerbrandy) [inline contracts/vaults/Deposit.sol:43]: let's not mix fixes, can you restore?

File: contracts/vaults/Deposit.sol [hunk 2]
```diff
@@ -97,6 +97,7 @@ contract Deposit is Base {
         // if the user already has funds in the pool, give the previous reward
         if (_user.shares > 0) {
             uint256 pending = _user.shares * poolInfos[_pid].rewardPerShare / 1e12 - _user.rewardDebt;
+            userInfo[_pid][msg.sender].rewardDebt += pending;
```
Comment (jellegerbrandy) [inline contracts/vaults/Deposit.sol:100]: pls let's not change this in this pr?

**Review by jellegerbrandy — CHANGES_REQUESTED** [review]

File: contracts/vaults/Deposit.sol (hunk 1 above)
Comment (ben-kaufman) [reply contracts/vaults/Deposit.sol]: Fixed

File: contracts/vaults/Deposit.sol (hunk 2 above)
Comment (ben-kaufman) [reply contracts/vaults/Deposit.sol]: Fixed, sorry must have left it there while working on the other stuff

**Review by jellegerbrandy — APPROVED** [review]

**Pull request merged**